### DISCUSSION TITLE
Make installation process more approachable for more restrictive environments.

### DISCUSSION THREAD

**Discussion by mskarbek** [Budibase/budibase#4285] [Help]
I would like to use self-hosted Budi, but I'm unable to use official docker images - they are based on Alpine, which is banned in my work environment. I would like to build my own images based on Red Hat UBI images - the only one that are allowed.
Is there any documentation describing how to achieve such thing?

**Comment by crashmaster18** [comment]
Agreed. One of the majors would work for compliance; Red Hat, SUSE or Ubuntu all have FIPS Validation, etc. This could even be a paid for enterprise feature, but compliance teams inside most international companies will limit what this software can be used for (or outright banned like above) without it...

**Comment by mjashanks** [comment]
@mskarbek @crashmaster18 fantastic feedback, thank you. We did not know this.

Totally possible for us to provide these new images, it's just a matter of priority. It's something that we would consider prioritising for enterprises that are serious about running Budibase in production.

Happy to chat about this further on a call https://calendly.com/mjashanks/budibase
